### ISSUE TITLE
v8.1 beta Communications List Has Missing Pieces

### ISSUE THREAD

**Issue by JimMichael** [SparkDevNetwork/Rock#3223]
I'm working through my process.st checklist and when I got to the task of creating a new Communications List synced to a DataView, I got stuck. For some reason, on my 8.1 server a BUNCH of stuff is missing from the create new list function. Instead of this
![v74 comm list](https://user-images.githubusercontent.com/3903408/44697193-40683480-aa40-11e8-95ed-2c049a79574b.png)
I see this:
![v81 comm list](https://user-images.githubusercontent.com/3903408/44697200-4827d900-aa40-11e8-99fd-6da35f7934c5.png)
Basically, everything beneath Group Attribute Values is missing... I would assume user error, but all I'm doing is going to Admin | Communications | Communications Lists and adding a new one.

**Comment by JimMichael** [comment]
I am seeing same on prealpha.

**Comment by nairdo** [comment]
@JimMichael Can you post a screenshot of your Communication List _type_ detail from v7.4 -- specifically the General section as seen in this screenshot:
![image](https://user-images.githubusercontent.com/383807/44726247-27857080-aa8c-11e8-8f01-3aa5eb35002a.png)

**Comment by JimMichael** [comment]
This is my v7.4 Communication List group type (General tab)
![image](https://user-images.githubusercontent.com/3903408/44728134-535f2100-aaa1-11e8-9267-420f3ba05e89.png)

My v8.1 group type looks the same, for what it's worth. I'm confused by your statement that this is now working as designed, though... without the _Group Sync Settings_ section on a Communication List
(v7.4 shown, missing in v8.1)
![image](https://user-images.githubusercontent.com/3903408/44728330-d08a9600-aaa1-11e8-8d54-3d48aabd1284.png)
how would you ever sync a DataView to a List in v8? Or are dynamic communication lists a deprecated thing now and they are only meant for static membership? I was just trying to follow the process.st instructions to create a new Comm List synced to a dataview, and in 7.4 I would have done this from the Communication List itself, but in v8 that seems impossible (unless there's another way I'm unaware of?)

**Comment by JimMichael** [comment]
Ok, I just realized that your screen shot above is v8, and there's this whole other section 
![image](https://user-images.githubusercontent.com/3903408/44728661-a84f6700-aaa2-11e8-8ffa-cb43c4cc9fe6.png)
in the group type on v8, and I assume that _Allow Group Sync_ setting is the culprit? Of so, I guess the only "error" here is that the end user won't know about that since it's a change from v7 behavior... so maybe it should be mentioned in the technical release notes? (that you will need to manually enable Group Sync on the Comms List group type if you want sync-able lists)

**Comment by nairdo** [comment]
Thanks @JimMichael I think that will do it -- I'll make sure there is a data migration to checkbox the "Allow Group Sync" setting on the Communication Lists group type so admins won't have to do this manually.
